### ISSUE TITLE
Update tools to support let* and while*

### ISSUE THREAD

**Issue by billzorn** [FPBench/FPBench#47]
- [ ] reader

- [ ] reference interpreter

- [ ] fpcore-extra

- [ ] canonicalizer / condenser

- [ ] core2c

- [ ] core2js

- [ ] core2sollya

- [ ] core2smt

- [ ] core2wls

- [ ] other exporters: gappa, fptaylor, go, scala, json?

**Comment by billzorn** [comment]
Some functionality in fpcore-extra has been updated, namely the transforms that are exposed in transform.rkt (unrolling, skipping loops, expanding let* and while*)

**Comment by bksaiki** [comment]
Ahh got it. That's why core2json is in infra. Good to know.

**Comment by pavpanchekha** [comment]
Yep. You could imagine a world where we have an FPCore parser in JS and then we don't need it—that would be so cool—but we don't right now.
